webapp/sources/rudder/rudder-web/src/main/webapp/techeditor/js/ncf.js

```diff
@@ -795,7 +795,6 @@ $scope.onImportFileChange = function (fileEl) {
             $scope.checkSelect(technique, $scope.selectTechnique);
             $scope.originalTechnique.bundle_name = undefined;
             $scope.originalTechnique.name = "";
-            $scope.ui.editForm.$setDirty();
```

Comment:
Can we stll save after this ?

Reply:
![test_import_technique](https://user-images.githubusercontent.com/23410978/105880673-ed757b80-6003-11eb-9c26-cd32170e2b93.gif)

Reply:
But the message ``` This Technique has been deleted while you were away. Saving it will recreate it. ``` is not normal